app/Http/Controllers/AdminDash.php

```diff
@@ -965,19 +969,22 @@ public function saveFeedback(Request $request, $id) {
         $audit->save();
 
         $controller = User::find($feedback->controller_id);
-
-        Mail::send(['html' => 'emails.new_feedback'], ['feedback' => $feedback, 'controller' => $controller], function ($m) use ($feedback, $controller) {
-            $m->from('feedback@notams.ztlartcc.org', 'vZTL ARTCC Feedback Department');
-            $m->subject('You Have New Feedback!');
-            $m->to($controller->email);
-        });
-
+        if (isset($controller)) {
+            Mail::send(['html' => 'emails.new_feedback'], ['feedback' => $feedback, 'controller' => $controller], function ($m) use ($feedback, $controller) {
```

Comment:
I'm worried about how this is checking whether this is assigned to a controller or event because we could have ID overlap in the future between the 2. Highly unlikely, but likely nonetheless especially if something happens to the ID incrementing or we have someone join with a very low CID...

I think it would be better to check this when it is being created and set a flag to indicate whether it's for an event or controller. On the other hand, we could also have 2 different columns that are nullable: `controller_id` and `event_id`. IMO it's not a good practice to mix ID's in one column and that's a pattern that could introduce a bunch of bugs that could be hard to find

We may want to think a little bit more on the best way of adding in events here, but I don't think we should go through with sharing a column for 2 different IDs